yarn.config.cjs

```diff
@@ -14,7 +15,7 @@ const { inspect } = require('util');
 /**
  * Aliases for the Yarn type definitions, to make the code more readable.
  *
- * @typedef {import('@yarnpkg/types').Yarn} Yarn
+ * @typedef {import('@yarnpkg/types').Yarn.Constraints.Yarn} Yarn
```

Comment:
Used `// @ts-check`, which indicated some JSDoc types that needed updating. Happy to revert.